src/pages/authorization/index.mdx

```diff
@@ -44,6 +44,13 @@ If you'd like to see the authorization flows explained live, watch our video tut
   allowfullscreen
 ></iframe>
 
+# Sample Authorization Apps
+
+We've prepared two apps which include fully implemented authorization code samples. You can test them, experiment with them, or modify one of this app’s code and extend its functionality.
+
```

Comment:
- <a href="https://github.com/livechat/sample-app-popup-auth" target="_blank">Sample app with the popup flow</a>
- <a href="https://github.com/livechat/sample-app-redirect-auth" target="_blank">Sample app with the redirect flow</a>

Reply:
Or maybe we could use the `<SectionLink>` component here? 👆🏻

Reply:
Tomorrow, I will see how it looks and decide:) However, it seems like a very good idea

Reply:
I have added normal links, as the `<SecrionLink>' component was not working when I tried. However, it still looks very similar to the `<SectionLink>' :)

Reply:
ok